os/board/common/crashdump.c

```diff
@@ -208,6 +210,68 @@ static int get_ext_userfs_part(size_t *size, int *index)
 	return 0;
 }
 
+static int send_file(void)
+{
+	int i;
+	int fd;
+	int ret;
+	struct stat st;
+	off_t size;
+	char buf;
+	char filename[CONFIG_SMARTFS_MAXNAMLEN + 1];
```

Comment:
I think its better to initialize the filename with null characters.

Reply:
Understood, will do